src/Symfony/Bundle/TwigBundle/CacheWarmer/TemplateCacheWarmer.php

```diff
@@ -26,21 +26,36 @@
  */
 class TemplateCacheWarmer implements CacheWarmerInterface, ServiceSubscriberInterface
 {
-    private ContainerInterface $container;
     private Environment $twig;
-    private iterable $iterator;
+    private string $cacheFolder;
 
-    public function __construct(ContainerInterface $container, iterable $iterator)
+    /**
+     * @param string $cacheFolder
+     */
+    public function __construct(private ContainerInterface $container, private iterable $iterator /* , private string $cacheFolder */)
```

Comment:
```suggestion
    public function __construct(private ContainerInterface $container, private iterable $iterator, private string $cacheFolder = 'twig')
```

Adding new optional parameters to a constructor is not a BC break.

Reply:
Oh, didn't realise it was not a BC break, corrected 👍